customer.go

```diff
@@ -17,75 +19,31 @@ type Customer struct {
 }
 
 // Create a customer
-func CreateCustomer(client *Client, data *Customer) {
-	values := map[string]interface{}{
-		"auth":         client.Auth,
-        "api_key":      client.APIKey,
-        "firstname":    data.Firstname,
-        "lastname":     data.Lastname,
-        "email":        data.Email,
-        "phone":        data.Phone,
-        "user_id":      data.UserId,
-        "notes":        data.Notes,
-        "location_id":  data.LocationId,
-	}
-
+func CreateCustomer(data *Customer) map[string]interface{} {
     endpoint := APIBaseUrl + "customers/create"
-
-    Response(values, endpoint)
+    return Response(data, endpoint)
 }
 
 // Retrieve a list of customers
-func RetrieveCustomers(client *Client) {
-    values := map[string]interface{}{
-		"auth": client.Auth,
-        "api_key": client.APIKey,
-    }
-
+func RetrieveCustomers(data *Customer) map[string]interface{} {
```

Comment:
Is there pagination on any of these bulk listing endpoints?

Retrieving 10,000 records at once would put a fair amount of strain on the server

Reply:
This is definitely a TODO and certainly needed, but the API does not currently support pagination currently. It's on the list - sadly this can't be addressed here on the client library.